src/nft/components/nftExploreBanner/NftExploreBanner.tsx

```diff
@@ -80,13 +81,23 @@ const StyledXButton = styled(X)`
 `
 
 const StyledImageContainer = styled(Box)`
-  width: 23%;
+  width: 20%;
   cursor: pointer;
   aspectratio: 1;
   transition: transform 0.25s ease 0s;
   object-fit: contain;
 `
 
+const LinkStyle = css`
+  color: ${({ theme }) => theme.accentActive};
+  stroke: ${({ theme }) => theme.accentActive};
+`
+
+const StyledLink = styled(Link)`
```

Comment:
nit: you could just use a `styled(StyledLink)` and simply define a custom color/stroke for it!

Reply:
not sure what "StyledLink" you are referring to? Also, we need ClickableStyle as well, not just the custom colors.